### PR TITLE
Feat: I have created the models and migrations to db

### PR DESCRIPTION
**Pull request by programmerjide**

### CODE REVIEW

File: Models/Appointment.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        type: DataTypes.TIME,
+      },
+      reason_for_appointment: {
+        type: DataTypes.ENUM,
+        values: ["consultation", "antenatal", "others"],
+      },
+      patient_id: {
+        type: DataTypes.INTEGER,
+      },
+      staff_id: {
```
Comment (ogunsoladebayo) [inline Models/Appointment.js:45]: please make this field more descriptive. You can change to 'attending_staff_id'

File: Models/HealthCareProvider.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        type: DataTypes.STRING,
+      },
+      hotline: {
+        type: DataTypes.STRING,
+      },
+      private_govt: {
+        type: DataTypes.ENUM,
+        values: ["private", "government"],
+      },
+      occupation: {
```
Comment (ogunsoladebayo) [inline Models/HealthCareProvider.js:37]: There should be no occupation field for Health care providers

**Review by ogunsoladebayo — CHANGES_REQUESTED** [review]
Great job. Please add 'allow null: false' to all the fields that should require it.
Thanks.

File: Models/HealthCareProvider.js (hunk 2 above)
Comment (programmerjide) [reply Models/HealthCareProvider.js:37]: Which of the fields? or do you mean everything?

**Pull request merged**